### ISSUE TITLE
Wait for user server to be available before printing that its available

### ISSUE THREAD

**Issue by yairhaimo** [wix/yoshi#523]
# 🙋 Feature Request

### 😯 Current Behavior

We print that the `Application is now available` before we even run the application server. 

### 🤔 Expected Behavior

Print the `Application is now available` message only after the server is really up and running.

### 💁 Possible Solution 

Listen to the server port and print the message only after you receive a response from it.
[wait-port](https://www.npmjs.com/package/wait-port) might be a viable solution.

### 🔦 Context 

Improve DX - If the user is quick enough and accesses the URL they will receive a "page not found" message.

**Comment by netanelgilad** [comment]
Merged into v4 in #538
